website/pages/docs/acl/auth-methods/oidc.mdx

```diff
@@ -0,0 +1,214 @@
+  all be stringified when returned.
+
+- `ListClaimMappings` `(map[string]string)` - Mappings of claims (key)
+  [will be copied to a metadata field](#trusted-identity-attributes-via-claim-mappings)
+  (value). Use this if the claim you are capturing is list-like (such as groups).
+
+  When mapped, the values in each list can be any of a number, string, or
+  boolean and will all be stringified when returned.
+
+- `OIDCScopes` `(array<string>)` - Comma-separated list of OIDC scopes.
+
+- `JWTSupportedAlgs` `(array<string>)` - JWTSupportedAlgs is a list of
+  supported signing algorithms. Defaults to `RS256`. ([Available
+  algorithms](https://github.com/hashicorp/consul/blob/master/vendor/github.com/coreos/go-oidc/jose.go#L7))
+
+- `BoundAudiences` `(array<string>)` - List of `aud` claims that are valid for
+  login; any match is sufficient.
+
+- `VerboseOIDCLogging` `(bool: false)` - Log received OIDC tokens and claims when
+  debug-level logging is active.  Not recommended in production since sensitive
+  information may be present in OIDC responses.
+
+### Sample Config
+
+```json
+{
+    ...other fields...
+    "Config": {
+        "AllowedRedirectURIs": [
+            "http://localhost:8550/oidc/callback",
+            "http://localhost:8500/ui/oidc/callback"
+        ],
+        "BoundAudiences": [
+            "V1RPi2MYptMV1RPi2MYptMV1RPi2MYpt"
+        ],
+        "ClaimMappings": {
+            "http://example.com/first_name": "first_name",
+            "http://example.com/last_name": "last_name"
+        },
+        "ListClaimMappings": {
+            "http://consul.com/groups": "groups"
+        },
+        "OIDCClientID": "V1RPi2MYptMV1RPi2MYptMV1RPi2MYpt",
+        "OIDCClientSecret": "...(omitted)...",
+        "OIDCDiscoveryURL": "https://my-corp-app-name.auth0.com/"
+    }
+}
+```
+
+## JWT Verification
+
+JWT signatures will be verified against public keys from the issuer via OIDC
+discovery. Keys will be fetched from the OIDC Discovery URL during
+authentication and OIDC validation criteria (e.g. `iss`, `aud`, etc.) will be
+applied.
+
+## OIDC Authentication
+
+Consul includes two built-in OIDC login flows: the Consul UI, and the CLI using
+[`consul login`](/docs/commands/login).
+
+### Redirect URIs
+
+An important part of OIDC auth method configuration is properly setting
+redirect URIs. This must be done both in Consul and with the OIDC provider, and
+these configurations must align. The redirect URIs are specified for an auth
+method with the [`AllowedRedirectURIs`](#allowedredirecturis) parameter. There
+are different redirect URIs to configure the Consul UI and CLI flows, so one or
+both will need to be set up depending on the installation.
+
+#### Consul UI
+
+Logging in via the Consul UI requires a redirect URI of the form:
+`http://localhost:8500/ui/oidc/callback` or
+`https://{host:port}/ui/oidc/callback`
+
+The "host:port" must be correct for the Consul server.
```

Comment:
IS this the Consul server that is serving the UI specifically? That might be good to clarify because we often recommend that it is _not_ one of the actual Server agents here but rather a client which makes this statement a little confusing.

Reply:
I'll change the wording to clarify this is the consul agent you are interacting with (the one serving the UI).